md/maintenance-operation.md

```diff
@@ -7,36 +7,35 @@ Archive tables contain the execution history of the Bonita Platform. Some of the
 
 ## Archive Contract Data
 
-Archive contract data table keeps the information of all contracts sent to execute tasks or instantiate processes. After of a lot of tasks or processes were executed, the `archive_contract_data` table could become very big and consume a lot of disk spaces.
+Archive contract data table keeps the information of all contracts sent to execute tasks or instantiate processes.
+After of a lot of tasks or processes were executed, the `archive_contract_data` table could become very big and
+consume a lot of disk space.
 
-Those information are not required by Bonita Platform to work so table could be purge to reduce disk space usage.
+Those information are not required by Bonita Platform to work, so the table could be purged to reduce disk space usage.
                                                                                                                                  
 
-These section will describe how to purge the `archive_contract_data` table, depending on the version of Bonita Platform installed.
+These section will describe how to purge the `archive_contract_data` table, depending on the version of Bonita Runtime installed.
 
 :::warning
-
-_This operation is destructive, data will be not usable for audit or analytic purpose. It is highly recommended to backup data before the purge._
+_This operation is destructive, data will be not usable for audit or analytics purposes. It is highly recommended to backup data
+before the purge._
 :::
 
-_Keep in mind that the contract data's will continue to be archived and the table will continue to be filled,  even after purge has been done. Hoverer, it's possible to deactivate the archiving of `contract data's`, please see ([Configurable Archive](configurable-archive.md).)_  
-   
+_Keep in mind that the contract data will continue to be archived and the table will continue to grow, even after the purge has been done.
+However, from version 7.10, it is possible to deactivate the archiving of `contract data`, please see ([Configurable Archive](configurable-archive.md).)_  
```

Comment:
You talk a bout 7.10 in 7.9 branch. Is it intended ?

Reply:
To push customer to migrate, maybe? ...